trimeshpy/vtk_util.py

```diff
@@ -180,50 +178,58 @@ def lines_to_vtk_polydata(lines, colors="RGB"):
 
     # Set Points to vtk array format
     vtk_points = numpy_to_vtk_points(points_array)
-    lines_array = ns.numpy_to_vtk(lines_array, array_type=vtk.VTK_INT)
 
     # Set Lines to vtk array format
     vtk_lines = vtk.vtkCellArray()
-    vtk_lines.SetNumberOfCells(nb_lines)
-    vtk_lines.GetData().DeepCopy(lines_array)
-
-    # colors test, todo improve
-    if colors is not None:
-        if colors == "RGB":  # set automatic rgb colors
-            colors = np.abs(lines[:, -1] - lines[:, 0])
-            colors = 0.8 * colors / \
-                np.sqrt(np.sum(colors ** 2, axis=1, keepdims=True))
-            colors_mapper = np.repeat(lines_range, points_per_line, axis=0)
-            vtk_colors = numpy_to_vtk_colors(255 * colors[colors_mapper])
-        else:
-            colors = np.array(colors)
-            if colors.dtype == np.object:  # colors is a list of colors
-                vtk_colors = numpy_to_vtk_colors(255 * np.vstack(colors))
-            else:
-                # one colors per points / colormap way
-                if len(colors) == nb_points:
-                    vtk_colors = ns.numpy_to_vtk(colors, deep=True)
-                    raise NotImplementedError()
-
-                elif colors.ndim == 1:  # the same colors for all points
-                    vtk_colors = numpy_to_vtk_colors(
-                        np.tile(255 * colors, (nb_points, 1)))
-
-                elif colors.ndim == 2:  # map color to each line
-                    colors_mapper = np.repeat(
-                        lines_range, points_per_line, axis=0)
-                    vtk_colors = numpy_to_vtk_colors(
-                        255 * colors[colors_mapper])
-
-        vtk_colors.SetName("Colors")
+    if vtk.VTK_MAJOR_VERSION <= 8:
```

Comment:
@arnaudbore 
Here is the to support both before  (VTK <=8) and now (VTK 9)
I tested in mi-brain and with  "save_polydata(..., legacy_vtk_format=True)" it should work.
Could you double check on your side ?

Reply:
I think I don't understand. Now that our requirements ask for vtk >=9.2.0 we can never have vtk.VTK_MAJOR_VERSION <= 8. 
I think the idea here is to know if we want the legacy way to save the fib or the new way knowing we use vtk >=9.2.*.

Reply:
EDIT: I misunderstood the question, sorry Arnaud (we figured it out offline)

His point was that there is no way to install (now trimeshpy with anything below VTK==9.* so that first part of the IF will never be accessible with installation via `pip install`

Is there a reason to keep it? Do you sometime revert to python 3.7 and install vtk8?

Reply:
Python 3.7 has been dropped last year...not really a good thing to keep supporting it, I think.

Reply:
@arnaudbore @frheault 
Can you test the updated version, after this it will be merge for trimeshpy 0.0.4, and afterward update the pypi package

Reply:
I just tested with your 2 new commit the Scilpy branch linked above, I successfully open a .vtk and a .fib in MI-Brain !